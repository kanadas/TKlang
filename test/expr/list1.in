//int_list = int * int_list + void
let append = (\x l -> 1@(x, l)) in append 5 (1@(4, 1@(3, 1@(2, 1@(1, 2@())))))
