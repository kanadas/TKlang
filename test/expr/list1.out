expr/list1.in
Not concrete type: ((int*((int*((int*((int*(#5+void))+#6))+#7))+#8))+#9)
